contrib/babelfishpg_tsql/src/format.c

```diff
@@ -59,7 +58,7 @@ format_datetime(PG_FUNCTION_ARGS)
 
 	culture = text_to_cstring(PG_GETARG_TEXT_P(2));
 
-	valid_culture = format_validate_and_culture(culture, "LC_TIME");
+	(void) format_validate_and_culture((char *) culture, "LC_TIME");
```

Comment:
Instead of casting these to (char *), can we change the function declaration to use const char * instead?